<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetViewer</class>
 <widget class="QWidget" name="NetViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QSplitter" name="spltMain">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,1">
       <item>
        <layout class="QGridLayout" name="notificationLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="lblOffline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;a href=&quot;server.offline&quot;&gt;Why is my relay offline?&lt;/a&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblConsensus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>If your relay isn't listed among the others, it may be because it doesn't have a Running flag yet. &lt;a href=&quot;server.consensus&quot;&gt;What's this?&lt;/a&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="spltMap">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_1">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frmMap">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>100</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>300</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLineEdit" name="lineRouterSearch">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="RouterListWidget" name="treeRouterList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>175</width>
               <height>0</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="indentation">
              <number>0</number>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>6</number>
             </property>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Relay</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IP</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Bandwidth</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Uptime</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="spltRouter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Exit circuits</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="CircuitListWidget" name="treeCircuitList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>100</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Connection</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Internal circuits</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="CircuitListWidget" name="treeInternalCircuitList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>100</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Connection</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Hidden Service circuits</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="CircuitListWidget" name="treeHSCircuitList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>100</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Connection</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="RouterDescriptorView" name="textRouterInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>140</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="visible">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="grpRelayPanel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>110</height>
         </size>
        </property>
        <property name="title">
         <string>Add columns to Relay panel:</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkShowIP">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkShowBW">
           <property name="text">
            <string>Bandwidth</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="chkShowUptime">
           <property name="text">
            <string>Uptime</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveSettings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>27</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Saves the current Network Map settings</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancelSettings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>27</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Cancels changes made to settings</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/view-refresh.png</normaloff>:/images/32x32/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh the list of Tor relays and connections</string>
   </property>
   <property name="statusTip">
    <string>Refresh the list of Tor relays and connections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/system-help.png</normaloff>:/images/32x32/system-help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Show the network map help</string>
   </property>
   <property name="statusTip">
    <string>Show network map help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/window-close.png</normaloff>:/images/32x32/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the network map</string>
   </property>
   <property name="statusTip">
    <string>Close the network map</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/zoom-in.png</normaloff>:/images/32x32/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in on the network map</string>
   </property>
   <property name="statusTip">
    <string>Zoom in on the network map</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/zoom-out.png</normaloff>:/images/32x32/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out on the network map</string>
   </property>
   <property name="statusTip">
    <string>Zoom out on the network map</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionZoomToFit">
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/zoom-fit-best.png</normaloff>:/images/32x32/zoom-fit-best.png</iconset>
   </property>
   <property name="text">
    <string>Zoom To Fit</string>
   </property>
   <property name="toolTip">
    <string>Zooms to fit all currently displayed circuits</string>
   </property>
   <property name="statusTip">
    <string>Zooms to fit all currently displayed circuits</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionZoomFullScreen">
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/view-fullscreen.png</normaloff>:/images/32x32/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="toolTip">
    <string>View the network map as a full screen window</string>
   </property>
   <property name="statusTip">
    <string>View the network map as a full screen window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/vidalia.qrc">
     <normaloff>:/images/32x32/preferences-other.png</normaloff>:/images/32x32/preferences-other.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Adjust Network Map Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RouterDescriptorView</class>
   <extends>QTextEdit</extends>
   <header>network/RouterDescriptorView.h</header>
  </customwidget>
  <customwidget>
   <class>RouterListWidget</class>
   <extends>QTreeWidget</extends>
   <header>network/RouterListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CircuitListWidget</class>
   <extends>QTreeWidget</extends>
   <header>network/CircuitListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/vidalia.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>frmSettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
